File: Iniciante/1020.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main() {

    int dias, mes, anos, resto;


    scanf("%d", &dias);


    anos = dias /365;
    resto = dias % 365;
    mes = resto / 30;
    resto = resto % 30;



    printf("%d ano(s)\n", anos);
    printf("%d mes(es)\n", mes);
    printf("%d dia(s)\n", resto);



    return 0;
}

File: Iniciante/Python/1020.py
dias_total = int(input()) #400

anos = dias_total // 365 
resto = dias_total % 365
mes = resto // 30
dias = resto % 30

print("{:.0f} ano(s)".format(anos))
print("{:.0f} mes(es)".format(mes))
print("{:.0f} dia(s)".format(dias))
